bocas/result.py

```diff
@@ -53,30 +59,67 @@ def get(self, name):
 
     @staticmethod
     def load(path):
-        with open(f"{path}/results.p", "rb") as f:
-            result = pickle.load(f)
+        # Maintain backwards compatibility with pickled results
+        if os.path.exists(f"{path}results.p"):
```

Comment:
is the `/` missing here?  I think we should either:

- use pathlib
- use os.path.join

if we are going to drop the `/`

Reply:
Agreed. Adjusted to work with os.path.join